docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -109,6 +109,179 @@ Finally, we invoke the `codegenNativeComponent` generic function, passing the na
+- `jsSrcsDir`: the relative path to access the `js` specification that will be parsed by the Codegen.
+
+#### iOS: Create the `podspec` file
+
+Now it's time to configure the native component so that we can generate the required code and we can include it in our app.
+
+For iOS, we need to create a `podspec` file which will define the component as a dependency.
+The `podspec` file for our component will look like this
+
+```ruby title="rnt-centered-text.podspec"
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+folly_version = '2021.06.28.00-v2'
+folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
+
+Pod::Spec.new do |s|
+  s.name            = "rnt-centered-text"
+  s.version         = package["version"]
+  s.summary         = package["description"]
+  s.description     = package["description"]
+  s.homepage        = package["homepage"]
+  s.license         = package["license"]
+  s.platforms       = { :ios => "11.0" }
+  s.author          = package["author"]
+  s.source          = { :git => package["repository"], :tag => "#{s.version}" }
+
+  s.source_files    = "ios/**/*.{h,m,mm,swift}"
+
+  s.dependency "React-Core"
+
+  s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
+  s.pod_target_xcconfig    = {
+    "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\"",
+    "OTHER_CPLUSPLUSFLAGS" => "-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1",
+    "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
+  }
+
+  s.dependency "React-RCTFabric"
+  s.dependency "React-Codegen"
+  s.dependency "RCT-Folly", folly_version
+  s.dependency "RCTRequired"
+  s.dependency "RCTTypeSafety"
+  s.dependency "ReactCommon/turbomodule/core"
+end
+```
+
+The `podspec` file has to be a sibling of the `package.json` file and its name is the one we set in the `package.json`'s `name` property: `rnt-centered-text`.
+
+The first part of the file prepares some variables we will use throughout the rest of it:
+
+- the `package` variable which contains information read from the `package.json`
+- the `folly_version` variable to set the version of `folly` we need.
+- the `folly compiler_flags` that we need to set in the new architecture.
+
+The next section contains some information used to configure the pod, like its name, version, a description and so on.
+
+Finally, we have a set of dependencies that are required by the new architecture.
+
+#### Android: Add the `build.gradle` file
+
+For what concerns Android, we need to create a `build.gradle` file in the `android` folder. The file will have the following shape
+
+```kotlin title="build.gradle"
+buildscript {
+  ext.safeExtGet = {prop, fallback ->
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+  }
+  repositories {
+    google()
+	  gradlePluginPortal()
+  }
+  dependencies {
+    classpath("com.android.tools.build:gradle:7.0.4")
```

Comment:
This is already older than what we have on `main` so maybe `7.2.0` works better here